src/Struct/FilterStruct.php

```diff
@@ -19,12 +21,32 @@ class FilterStruct {
     public $value;
 
     public function __construct($fieldName, $operator, $value) {
-        $this->fieldName = $fieldName;
+        if (strpos($fieldName, self::$LOGICAL_OR)) {
+            $this->fieldName = explode(self::$LOGICAL_OR, $fieldName);
+        } else {
+            $this->fieldName = $fieldName;
+        }
         $this->operator = $this->transformOperator($operator);
         $this->value = $value;
     }
 
     public function apply($object) {
+        if (is_array($this->fieldName)) {
+            $me = $this;
+
+            $subquery = function ($query) use ($me) {
+                $query = $query->where($me->fieldName[0], $me->operator, $me->value);
+
+                for ($fieldIndex = 1; $fieldIndex < count($me->fieldName); $fieldIndex++) {
+                    $query = $query->orWhere($me->fieldName[$fieldIndex], $me->operator, $me->value);
```

Comment:
After delete `$me`, you can delete `use ($me)` and use `$this` instead of `$me`
```
            $subquery = function ($query) {
                $query = $query->where($this->fieldName[0], $this->operator, $this->value);

                for ($fieldIndex = 1; $fieldIndex < count($this->fieldName); $fieldIndex++) {
                    $query = $query->orWhere($this->fieldName[$fieldIndex], $this->operator, $this->value);
```